lib/cacheconfiguration.cpp

```diff
@@ -135,6 +167,45 @@ void CacheConfigurationPrivate::onGroupPropertyChanged()
 }
 #endif
 
+#ifdef HAS_QGSETTINGS
+void CacheConfigurationPrivate::onConfigPropertyChanged(const QString &key) {
```

Comment:
Nitpick: by convention the method starting braces on their own lines but I can adjust.